evalml/tests/model_understanding_tests/test_partial_dependence.py

```diff
@@ -287,6 +287,45 @@ def test_partial_dependence_multiclass(logistic_regression_multiclass_pipeline_c
     assert len(two_way_part_dep.columns) == grid_resolution + 1
 
 
+def test_partial_dependence_multiclass_numeric_labels(
```

Comment:
@freddyaboulton I'm sure you might have some opinions about me adding yet another long running test 😂 I wasn't confident in a way to mock things but definitely would love suggestions/opinions!

Reply:
What's the runtime now? My takeaway is that using `grid_resolution=5` can trim the test time to a couple seconds down from like ~10. The next step would be using fewer rows.

Reply:
~30-40 seconds before, 8 seconds after setting grid_resolution to 5 😁